x-pack/plugins/snapshot_restore/public/app/sections/home/repository_list/repository_details/repository_details.tsx

```diff
@@ -227,12 +229,19 @@ const RepositoryDetailsUi: React.FunctionComponent<Props> = ({
             <EuiFlexGroup alignItems="center">
               <EuiFlexItem grow={false}>
                 <RepositoryDeleteProvider>
-                  {(deleteRepository: (names: Array<Repository['name']>) => void) => {
+                  {(
+                    confirmDeleteRepository: (
```

Comment:
I find this name a bit confusing because I imagine that the _modal_ is where the user _confirms_ the delete action.  But if I understand the code correctly, we're using `confirmDeleteRepository` to open the confirmation modal. Can we rename this to communicate that we're prompting the user for confirmation? How about `promptUserToDeleteRepository` or simply `promptUser` since the context of "deleting repositories" is already established by the component name?